### PR TITLE
docs: Created docs for rule 'component-class-suffix'

### PR DESCRIPTION
**Pull request by gSarciotto**
Docs for rule [component-class-sufix](https://github.com/angular-eslint/angular-eslint/blob/master/packages/eslint-plugin/src/rules/component-class-suffix.ts), in accordance to the effort for documentation #197

### CODE REVIEW

File: packages/eslint-plugin/docs/rules/component-class-suffix.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+This rule enforces that classes decorated with @Component must have suffix "Component" (or custom) in their name.
+
+## Rule Details
+
+This rule follows the recommendation from the [Angular styleguide](https://angular.io/guide/styleguide#style-02-03).
+
+Examples of **incorrect** code for this rule (with default configuration):
+
+```javascript
```
Comment (JamesHenry) [inline packages/eslint-plugin/docs/rules/component-class-suffix.md:11]: These should all be ` ```ts `, rather than ` ```javascript`

**Review by JamesHenry — CHANGES_REQUESTED** [review]
Thanks a lot for kicking off this effort! See the one comment (which applies multiple times)

File: packages/eslint-plugin/docs/rules/component-class-suffix.md (hunk 1 above)
Comment (gSarciotto) [reply packages/eslint-plugin/docs/rules/component-class-suffix.md]: Oh, I completely missed it haha, thanks!

**Review by JamesHenry — APPROVED** [review]
Thanks!

**Pull request merged**